docs/getting-started/configuration.rst

```diff
@@ -21,124 +21,415 @@ Below are all the available options, simple examples can be found in the docker-
 
 .. _docker-sync-boilerplate: https://github.com/EugenMayer/docker-sync-boilerplate
 
-References
-----------
 
-.. caution::
-
-    This is a configuration reference. **Do not use all options at once, they do not make sense!** Or copy them as your starting point, rather use the docker-sync-boilerplate and then cherry pick the options you need.
+Simple configuration file
+-------------------------
 
 .. code-block:: yaml
 
     version: "2"
     options:
-      # default: docker-compose.yml if you like, you can set a custom location (path) of your compose file like ~/app/compose.yml
-      # HINT: you can also use this as an array to define several compose files to include. Order is important!
-      compose-file-path: 'docker-compose.yml'
+      verbose: true
+    syncs:
+      appcode-native-osx-sync:
+        src: './app'
+        sync_excludes: ['ignored_folder', '.ignored_dot_folder']
 
-      # optional, default: docker-compose-dev.yml if you like, you can set a custom location (path) of your compose file. Do not set it, if you do not want to use it at all
 
-      # if its there, it gets used, if you name it explicitly, it HAS to exist
-      # HINT: you can also use this as an array to define several compose files to include. Order is important!
-      compose-dev-file-path: 'docker-compose-dev.yml'
+General options
+---------------
+Configuration options below the top-level ``options`` key.
 
-      # optional, activate this if you need to debug something, default is false
-      # IMPORTANT: do not run stable with this, it creates a memory leak, turn off verbose when you are done testing
-      verbose: false
 
-      # ADVANCED: the image to use for the rsync container. Do not change this until you exactly know, what you are doing
-       # replace <sync_strategy> with either rsync, unison, native_osx to set a custom image for all sync of this type
-       # do not do that if you really do not need that!
-      <sync_strategy>_image: 'yourcustomimage'
+cli_mode
+^^^^^^^^
+Defines how the sync will be invoked on the command line.
+Mostly depending if your are using docker-sync solo,
+scaffolded or in development (thor).
 
-      # optional, default auto, can be docker-sync, thor or auto and defines how the sync will be invoked on the cli. Mostly depending if your are using docker-sync solo, scaffolded or in development ( thor )
-      cli_mode: 'auto'
-      # optional, maximum number of attempts for unison waiting for the success exit status. The default is 5 attempts (1-second sleep for each attempt). Only used in unison.
-      max_attempt: 5
+Optional.
 
-      # optional, default: pwd, root directory to be used when transforming sync src into absolute path, accepted values: pwd (current working directory), config_path (the directory where docker-sync.yml is found)
-      project_root: 'pwd'
+default:
+  ``auto``
+
+possible values:
+  - ``docker-sync``
+  - ``thor``
+  - ``auto``
```

Comment:
maybe we should use a different format here, e.g. a table with 2 columns,first col for the possible values, the second is the default? what is your opinion?

Reply:
I use the "possible values" to explain the option sometimes, so having them separate is helpful IMO

Reply:
sure, its more about table vs list